### DISCUSSION TITLE
`ignorePrAuthor`'s value from repo config is not being used by bot

### DISCUSSION THREAD

**Discussion by RahulGautamSingh** [renovatebot/renovate#29497] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

https://github.com/renovatebot/renovate/blob/3741d2a7df93532d91c1e2504d23a3ed61eb8217/lib/workers/repository/init/index.ts#L53-L55

`ignorePrAuthor` is set in the local config of a platform before repo config is read, which means that we always set the value of `ignorePrAuthor` from global config or the default value.

But it is documented as a option which can be present in both configs. Should we convert it to a global config option? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
I think it makes sense to be global-only
